python/mxnet/gluon/trainer.py

```diff
@@ -49,6 +51,9 @@ class Trainer(object):
         on the type of compression being used. For example, 2bit compression requires a threshold.
         Arguments would then be {'type':'2bit', 'threshold':0.5}
         See mxnet.KVStore.set_gradient_compression method for more details on gradient compression.
+    update_on_kvstore : bool, default None
+        Whether to force the update to happen on kvstore. If None, then parameter update won't happen
```

Comment:
nit: Is "force" the right word? False means update won't happen on kvstore, instead of "not forcing update to happen on kvstore". 
What about "Whether to perform parameter updates on kvstore"?